chapter01/1.1 - Is Unique/isUnique.js

```diff
@@ -23,6 +23,23 @@ const everyCharUnique = (str, indexOffset = 'a'.charCodeAt()) => {
     return true;
 };
 
+function uniqueCharacter(str) {
+  let obj = {};
+  for (let i = 0; i < str.length; i++) {
+    if (obj[str[i]]) {
+      obj[str[i]] += 1;
```

Comment:
Why not exit at this point here, to save you from the extra for-loop checking for anything >1?

Reply:
Thanks updated the code, this reminds me of there is some space for optimising code.